src/gen/gen_help_html.lua

```diff
@@ -132,6 +130,8 @@ local exclude_invalid_urls = {
   ['http://wiki.services.openoffice.org/wiki/Dictionaries'] = 'spell.txt',
   ['http://www.adapower.com'] = 'ft_ada.txt',
   ['http://www.jclark.com/'] = 'quickfix.txt',
+  ['https://cacm.acm.org/research/a-look-at-the-design-of-lua/'] = 'faq.txt', -- blocks curl
+  ['https://linux.die.net/man/2/poll'] = 'luvref.txt', -- blocks curl
```

Comment:
I think it's more like it blocks azure or something. curl works for these, locally.